README.md

```diff
@@ -49,24 +49,37 @@ Refer to our GitHub pages for the course materials and additional resources:
 
 ## Requirements
 
-First we need to get pipenv installed. The best way is to install it for
+First we need to make sure we have pipenv installed:
+
+    pipenv --version
+
+If you don't have it installed, the best way is to install it only for
 your user:
 
     pip install --user pipenv
 
-Now we can use pipenv to install the rest of the dependencies, and
-create a virtual environment:
+## Getting started
+
+Use pipenv to create a virtual environment and to install the rest of the dependencies:
 
-    pipenv install
+    pipenv --python /usr/local/bin/python3.7 install
 
 You can also install packages from your distribution, but they may be
 too old.
 
+After creating the environment, we want to activate and enter our environment: 
+
+    pipenv shell
+
+Indication that you are inside the environment, the prompt line will look like this:
+
+    (ROSE) [student@vm29 ROSE]$
```

Comment:
this I think I would change because it's relevant for our specific VMs
(ROSE) [student@vm29 ROSE]$
maybe we can change it to:
(ROSE) [username@hostname ROSE]$